kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/tasks/graal/createGraalNativeImage.kt

```diff
@@ -72,7 +69,7 @@ fun Project.createGraalNativeImageTask(
 
                         godotJvmExtension.nativeImageToolPath.get(),
                         "-cp",
-                        "\"${godotBootstrapJar.absolutePath}\";\"${mainJar.absolutePath}\"",
+                        "\"${usercodeJar.absolutePath}\"",
```

Comment:
I think there's no more need of string template.

Reply:
For windows it is. There the `"` around the path are needed it seems